### ISSUE TITLE
Should detect tabs between tokens as invalid

### ISSUE THREAD

**Issue by TJM** [Pryz/yaml-lint#17]
I recently pasted some options from my sshd_config file and then added colons to the "key" side so that I could pass them to ssh::server_options: hash... however, one of the options had a "tab" character (\t) between the key and the value. Puppet balked with the following error:

```
.../hieradata/common.yaml): found character '\t(TAB)' that cannot start any token. (Do not use \t(TAB) for indentation) while scanning for the next token at line 509 column 22 on node
```

... but yaml-lint did not catch it. It should probably catch tabs as invalid whitespace characters between "tokens"

Example yaml:

```
ssh::server_options:
  AuthorizedKeysFile:	.ssh/authorized_keys
```

**Comment by TJM** [comment]
Actually, I think this problem exists elsewhere, as per specs, tab is OK as a separation character, but its being detected as an indent character (which is indeed not valid). http://yaml.org/spec/1.2/spec.html#id2778241
